File: 백준/Bronze/2566. 최댓값/최댓값.py
def solution(data_list):
    # 최댓값
    max_num = -1
    # 최대 인덱스
    max_idx = []
    # 순회
    for i in range(9):
        for j in range(9):
            # 현재 값이 최댓값보다 크면 최신화
            if data_list[i][j] > max_num:
                # 최댓값
                max_num = data_list[i][j]
                # 최대 인덱스
                max_idx = [i+1, j+1]
    print(max_num)
    print(*max_idx)
    
    
data_list = [list(map(int, input().split())) for _ in range(9)]
solution(data_list)

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <iostream>
#include <algorithm>

using namespace std;

int find_max(int data_list[9], int &max_num, int &max_idx)
{
	for (int i = 0; i < 9; i++)
	{
		if (max_num < data_list[i])
		{
			max_num = data_list[i];
			max_idx = i;
		}
	}
	return 0;
}


int main(void)
{
	int data_list[9] = {};
	int max_num = 0;
	int max_idx = 0;

	for (int i = 0; i < 9; i++)
	{
		cin >> data_list[i];
	}

	find_max(data_list, max_num, max_idx);
	cout << max_num << endl << max_idx+1 << endl;
}